src/ReviewService.Tests/ReviewServiceShould.cs

```diff
@@ -0,0 +1,64 @@
+using Moq;
+using ReviewService.Abstractions;
+
+namespace ReviewService.Tests;
+
+/// <summary>
+/// Tests for <see cref="ReviewService{TReviewSettings}"/>.
+/// </summary>
+public sealed class ReviewServiceShould
+{
+	private readonly Mock<IReviewPrompter> _reviewPrompterMock;
+	private readonly Mock<IReviewSettingsSource<ReviewSettings>> _reviewSettingsSourceMock;
+
+	public static IList<object[]> ReviewSettingsAndConditionsSatisfiedMapping =>
+		new List<object[]>
+		{
+			new object[]
+			{ 
+				new ReviewSettings(),
+				false
+			},
+			new object[]
+			{ 
+				new ReviewSettings
+				{
+					ApplicationFirstLaunched = DateTimeOffset.MinValue,
+					ApplicationLaunchCount = int.MaxValue,
+					PrimaryActionCompletedCount = int.MaxValue
+				},
+				true
+			}
+		};
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="ReviewServiceShould"/> class.
+	/// </summary>
+	public ReviewServiceShould()
+	{
+		_reviewPrompterMock = new Mock<IReviewPrompter>();
```

Comment:
why creating the mock in the ctor and not in the test methods

Reply:
I figured it would be cleaner and reusable in the future.